README.md

```diff
@@ -684,6 +684,20 @@ With this option enabled only environment variables starting with
 i.e. The environment variable `CONFIG_FORCE_a_b__c___d` set the
 configuration key `a.b-c_d`
 
+### Set array values from env variables
+
+Setting the value of array items from environment variables requires
```

Comment:
You mean a system property, not environment variables, right, from the example you give at the end?

Reply:
You're right, I fixed one occurrence but not the other.

Reply:
raised #710